### PR TITLE
3Sum Java Solution added

### PR DESCRIPTION
**Pull request by Arijit1000**

### CODE REVIEW

**Comment by Arijit1000** [comment]
Updated solution of merge k sorted Lists Problem.

File: coding.json [hunk 1]
```diff
@@ -1,26 +1,5 @@
 {
   "questions": [
-    {
```
Comment (marshallshen) [inline coding.json:3]: Can you undo the change to `coding.json`, the file will be automatically generated by [this Github action after merge](https://github.com/HackerPen/tech-interview-questions/blob/main/.github/workflows/generate_coding_json.yml)

File: coding/merge_k_sorted_lists/starter.java [hunk 2]
```diff
@@ -10,6 +10,26 @@
  */
 class Solution {
   public ListNode mergeKLists(ListNode[] lists) {
+    
+     if (lists == null || lists.length == 0) {
```
Comment (marshallshen) [inline coding/merge_k_sorted_lists/starter.java:14]: Can you put your solution code in the `SOLUTION_en.md` instead ([link](https://github.com/HackerPen/tech-interview-questions/blob/main/coding/merge_k_sorted_lists/SOLUTION_en.md))

File: coding/three_sum/starter.java [hunk 3]
```diff
@@ -1,5 +1,34 @@
 class Solution {
-  public List<List<Integer>> threeSum(int[] nums) {
-
-  }
+    public List<List<Integer>> threeSum(int[] nums) {
```
Comment (marshallshen) [inline coding/three_sum/starter.java:2]: Same comment as above, put your solution code in the `SOLUTION_en.md` instead ([link](https://github.com/HackerPen/tech-interview-questions/blob/main/coding/three_sum/SOLUTION_en.md))

File: system_design.json [hunk 4]
```diff
@@ -30,5 +30,5 @@
       ]
```
Comment (marshallshen) [inline system_design.json:30]: Undo this change.

**Review by marshallshen — COMMENTED** [review]
I added a few comments, thanks for contributing!

File: coding/merge_k_sorted_lists/starter.java (hunk 2 above)
Comment (Arijit1000) [reply coding/merge_k_sorted_lists/starter.java:14]: Sure, I will do so

**Comment by marshallshen** [comment]
@Arijit1000: let me know how this goes? You need to:

- rebase with `main` branch
- undo the change to `coding.json`
- put your solutions inside `SOLUTION_en.md` instead
